src/View/layout.html.twig

```diff
@@ -44,7 +44,7 @@
 								<a class="nav-link link-dark active" aria-current="page" href="#">Mon compte</a>
 							</li>
 							<li class="mb-1">
-								<a class="nav-link link-dark active" aria-current="page" href="#">A propos</a>
+								<a class="nav-link link-dark active" aria-current="page" href="/../contact">A propos</a>
```

Comment:
the double dot in the href is not needed, you should just have /contact

Reply:
double dot removed in the href